api/conf/conf.json

```diff
@@ -10,6 +10,17 @@
     "dag-lib-path": "",
     "etcd": {
       "endpoints": "127.0.0.1:2379"
+    },
+    "log": {
+      "error_log": {
+        "level": "warn",
+        "file_path": ""
+      },
+      "rotate": {
```

Comment:
can we implement this feature later? 
so we can remove those configurations at first.

Reply:
fair enough

Reply:
I can not get your mind

Reply:
It means OK.